### ISSUE TITLE
Error on build: opts.ShowFeedNames undefined 

### ISSUE THREAD

**Issue by williamla** [guyfedwards/nom#16]
Hi there, I was looking to create a quick docker for this but ran into issues using the Makefile when running build. this is on alpine with the latest version of go 1.19.4. Any ideas?

```
/nom # make build
go build -o nom cmd/nom/main.go
go: downloading github.com/jessevdk/go-flags v1.4.0
go: downloading github.com/charmbracelet/bubbles v0.14.0
go: downloading github.com/charmbracelet/bubbletea v0.23.1
go: downloading github.com/charmbracelet/lipgloss v0.6.0
go: downloading golang.org/x/term v0.0.0-20210927222741-03fcf44c2211
go: downloading gopkg.in/yaml.v3 v3.0.1
go: downloading github.com/JohannesKaufmann/html-to-markdown v1.3.6
go: downloading github.com/charmbracelet/glamour v0.5.0
go: downloading github.com/muesli/reflow v0.3.0
go: downloading github.com/sahilm/fuzzy v0.1.0
go: downloading github.com/containerd/console v1.0.3
go: downloading github.com/mattn/go-isatty v0.0.16
go: downloading github.com/mattn/go-localereader v0.0.1
go: downloading github.com/muesli/ansi v0.0.0-20211018074035-2e021307bc4b
go: downloading github.com/muesli/cancelreader v0.2.2
go: downloading github.com/muesli/termenv v0.13.0
go: downloading github.com/mattn/go-runewidth v0.0.14
go: downloading golang.org/x/sys v0.0.0-20220811171246-fbc7d0a398ab
go: downloading github.com/PuerkitoBio/goquery v1.8.0
go: downloading golang.org/x/net v0.0.0-20220909164309-bea034e7d591
go: downloading github.com/atotto/clipboard v0.1.4
go: downloading github.com/yuin/goldmark v1.4.14
go: downloading github.com/yuin/goldmark-emoji v1.0.1
go: downloading github.com/aymanbagabas/go-osc52 v1.0.3
go: downloading github.com/lucasb-eyer/go-colorful v1.2.0
go: downloading github.com/rivo/uniseg v0.2.0
go: downloading github.com/andybalholm/cascadia v1.3.1
go: downloading github.com/alecthomas/chroma v0.10.0
go: downloading github.com/microcosm-cc/bluemonday v1.0.17
go: downloading github.com/olekukonko/tablewriter v0.0.5
go: downloading github.com/dlclark/regexp2 v1.4.0
go: downloading github.com/aymerick/douceur v0.2.0
go: downloading github.com/gorilla/css v1.0.0
# command-line-arguments
cmd/nom/main.go:26:73: opts.ShowFeedNames undefined (type Options has no field or method ShowFeedNames)
make: *** [Makefile:2: build] Error 2
```

**Comment by guyfedwards** [comment]
Thanks for reporting, really need some CI set up. Fixed in [31d0e34](https://github.com/guyfedwards/nom/commit/31d0e34bef8a564825fc483e7943e77c5a1a81bb)

**Comment by williamla** [comment]
thanks. i'll try to set this up over the next few!
